tools/querytee/proxy_backend.go

```diff
@@ -81,7 +81,7 @@ func (b *ProxyBackend) createBackendRequest(orig *http.Request, body io.ReadClos
 	req.URL.Path = path.Join(b.endpoint.Path, req.URL.Path)
 
 	// Set the correct host header for the backend
-	req.Header.Set("Host", b.endpoint.Host)
```

Comment:
Shouldn't we  keep overriding the header too because it may be set in the orig request?

Reply:
`req.Host` takes precedence over that anyways https://github.com/golang/go/blob/master/src/net/http/request.go#L239-L243

```
	// For client requests, Host optionally overrides the Host
	// header to send. If empty, the Request.Write method uses
	// the value of URL.Host. Host may contain an international
	// domain name.
	Host string
```

Reply:
Ah, all good then, thanks!